server/src/main/java/io/littlehorse/server/streamsimpl/storeinternals/GetableStorageManager.java

```diff
@@ -175,7 +178,7 @@ private <U extends Message, T extends Getable<U>> void insertIntoStore(
 
     private <U extends Message, T extends Getable<U>> void deleteFromStore(
         String key,
-        StoredGetable<U, T> getable
+        StoredGetable<?, ?> getable
```

Comment:
If you do `StoredGetable<?, ?>`, this method (`deleteFromStore`) doesn't need to use generics anymore, so we should be able to get rid off `<U extends Message, T extends Getable<U>>`

Reply:
If we don't put the `<?, ?>` then VSCode gives a warning ):

Reply:
I meant to move from 

`private <U extends Message, T extends Getable<U>> void deleteFromStore(....) {}`

to:

`private void deleteFromStore(...) {}`